src/OpenTracing/Tracer.php

```diff
@@ -35,7 +35,7 @@
      */
     public function startSpan(
         $operationName,
-        SpanReference $parentReference = null,
+        Reference $parentReference = null,
         $startTimestamp = null,
         array $tags = []
     );
```

Comment:
that's an odd one - in Go StartWithOption was necessary because it does not support method overloading. But in Python you can do `startSpan("op")` or `startSpan("op", {..options..})`. Since PHP allows defaulting options to null, what's the reason for two method names?